bin/yugabyted

```diff
@@ -1769,52 +1821,204 @@ class YBAdminProxy(object):
+            env={"PGUSER": self.username, "PGPASSWORD": self.password, "PGDATABASE": self.db})
+        out, err = p.communicate()
+        return out, err
+
+    # Change specified DB's owner
+    def db_owner(self, db, new_owner):
+        cmd = [self.path, "-h", self.ip, "-p", self.port, "-c",
+                "alter database {} owner to {};".format(db, new_owner)]
+        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+            env={"PGUSER": self.username, "PGPASSWORD": self.password, "PGDATABASE": self.db})
+        out, err = p.communicate()
+        return out, err
+
+    # Check YSQL is UP
+    def is_ysql_up(self):
+        cmd = [self.path, "-h", self.ip, "-p", self.port, "-c", "\\conninfo"]
+        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+            env={"PGUSER": self.username, "PGPASSWORD": self.password})
+        out, err = p.communicate()
+        if err:
+            Output.log("Unable to connect using YSQL. Error: {}".format(err))
+            raise RetryableError()
+        else:
+            Output.log("YSQL Connection Info - {}".format(out))
+            return True
+
+# Proxy for ycqlsh commands.
+class YcqlProxy(object):
+    def __init__(self, ip, port, path=os.path.join(BIN_DIR, YUGABYTE_API_CLIENT_PROGRAMS["ycql"]),
+                    use_env_credentials=True):
+        self.ip = str(ip)
+        self.port = str(port)
+        self.path = path
+        self.username, self.password, self.keyspace = get_env_credentials(
+                                                        YUGABYTE_API_CLIENT_PROGRAMS["ycql"],
+                                                        use_env_credentials)
+
+    # Starts interactive YCQL shell.
+    def connect(self):
+        cmd = [self.path, self.ip, self.port, "-u", self.username]
+        if self.keyspace is not None:
+            cmd.extend(["-k", self.keyspace])
+        shell = subprocess.Popen(cmd)
+        shell.communicate()
+
+    # Check user exists
+    # Note that this will return false if ycqlsh can't connect, even if user exists.
+    def user_exists(self, ycql_username):
+        cmd = [self.path, self.ip, self.port, "-u", self.username, "-p", self.password, "-e",
+            "SELECT role FROM system_auth.roles WHERE role='{}';".format(ycql_username)]
+        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = p.communicate()
+        return out.strip() == ycql_username
+
+    # Create specified superuser
+    def create_user(self, ycql_username, ycql_password):
+        cmd = [self.path, self.ip, self.port, "-u", self.username, "-p", self.password, "-e",
+                "CREATE ROLE {} WITH PASSWORD = '{}' AND LOGIN = true AND SUPERUSER = true;".
+                format(ycql_username, ycql_password)]
+        p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
like the SQL case, it would be good to have a utility method to encapsulate running a cql command.